### ISSUE TITLE
Side effect of implicit `create` in `snapshot` subcommand

### ISSUE THREAD

**Issue by Juerd** [kopia/kopia#2823]
```
# kopia snapshot hlep 
Snapshotting root@hostname:/tmp/hlep ...
ERROR unable to get local filesystem entry: resolveSymlink: stat: lstat /tmp/hlep: no such file or directory
```

I mistyped `help` and it tried to create a snapshot :)

I'd like to suggest deprecating the implicit `create` subcommand, so that mistyped subcommands can get proper error messages.

**Comment by jkowalski** [comment]
I like the idea, I don't think we ever advertised the shortcut and it's actively confusing (say you had a directory called "help"), we should remove this (and possibly review other default subcommand and remove some more), mark it as a breaking change.

PR welcome.

**Comment by julio-lopez** [comment]
Addressed in #2861
